io/http/server/simplified endpoint listener/cf29d613-2fc6-4230-8c7f-bd86cb1c4c9e.json

```diff
@@ -55,13 +55,51 @@
           ],
           "name": "get timeout or default",
           "prototype": "46cb0e70-0d7b-44ed-8dd2-f94cf07995c9"
+        },
+        "f4364127-acb4-4ca2-90ba-9627fe77322e": {
+          "coordinates": [
+            196.5,
+            132
+          ],
+          "name": "check server error",
+          "prototype": "0faadab5-12bc-4bd8-b7e1-0bd8ddf21a43"
+        },
+        "f8c63529-af06-4f82-a8df-a382379413da": {
+          "coordinates": [
+            362.5,
+            258
+          ],
+          "name": "log server error",
+          "prototype": "fafa45e5-b1dd-4348-bcd8-ac6fb25b3d07",
+          "params": {
+            "5773699e-0819-4c74-8282-fb2a9a77b0cf": "\"[SERVER ERROR]\""
+          }
+        },
+        "ae9daab2-fa97-4956-8de8-a19f6306673a": {
+          "coordinates": [
+            210.16666666666669,
+            -251.66666666666663
+          ],
+          "name": "format url",
+          "prototype": "8f7f2e0d-2a80-428c-904b-c8d603fab102"
+        },
+        "4b2463ea-fab7-4a1e-9ec9-04ef9ae2016b": {
+          "coordinates": [
+            334.5,
+            -315
+          ],
+          "name": "log url",
```

Comment:
This is not logging an URL, but the fact that the server started. The name should reflect that